### PR TITLE
Workload tests adapted to centralized result logging

### PR DESCRIPTION
**Pull request by martin-mat**
## Description
Workload tests adapted to cenralized result logging

## Issues:
Refs: #1861 

## How has this been tested:
 - [ ] Covered by existing integration testing
 - [ ] Added integration testing to cover
 - [ ] Verified all A/C passes
     * [ ] develop
     * [ ] master
     * [ ] tag/other branch
 - [ ] Test environment
    * [ ] Shared Packet K8s cluster
    * [ ] New Packet K8s cluster
    * [ ] Kind cluster
 - [ ] Have not tested

## Types of changes:
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] Documentation update

## Checklist:
**Documentation**
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] No updates required.

**Code Review**
- [ ] Does the test handle fatal exceptions, ie. rescue block

**Issue**
- [ ] Tasks in issue are checked off

### CODE REVIEW

**Comment by martin-mat** [comment]
spec test result the same as for main branch (still some failures but those don't not seem to be caused by this change).
Ready for review.

**Comment by agentpoyo** [comment]
> spec test result the same as for main branch (still some failures but those don't not seem to be caused by this change). Ready for review.

We'll try to pull down and push up to verify github actions with no issues and manual testing prior to merging.

File: src/tasks/workload/5g_validator.cr [hunk 1]
```diff
@@ -17,10 +17,7 @@ end
 desc "Test if a 5G core is valid"
 task "smf_upf_core_validator" do |t, args|
   #todo change to 5g_core_validator
-  CNFManager::Task.task_runner(args) do |args, config|
-    task_start_time = Time.utc
-    testsuite_task = "smf_upf_core_validator"
-    Log.for(testsuite_task).info { "Starting test" }
+  CNFManager::Task.task_runner(args, task: t) do |args, config|
```
[the remark this answers is not in the document]
Comment (martin-mat) [reply src/tasks/workload/5g_validator.cr:20]: start time of every test case moved to the task runner
https://github.com/cnti-testcatalog/testsuite/blob/main/src/tasks/utils/task.cr#L113

and reported along with test case result the same way as before the changes 
https://github.com/cnti-testcatalog/testsuite/blob/main/src/tasks/utils/task.cr#L121

I'm not aware that test case duration/end time is recorded anywhere. But possible to introduce easily if needed.

**Comment by martin-mat** [comment]
rebased

**Comment by HashNuke** [comment]
Just tried running the build for this. There are some tests that are failing on repeated attempts.

https://github.com/cnti-testcatalog/testsuite/actions/runs/8232211049

**Comment by martin-mat** [comment]
fixed.
@HashNuke in the link provided I saw following failures of the spec tests:
curl, helm, operator_test, points , validate_config

I fixed reasons for: helm, operator_test, points , validate_config.
Failing "curl" specs: I don't know, I think it is not related to this change. For me it is failing even on main branch.

Can you please re-run github actions? Thanks.

**Comment by HashNuke** [comment]
* LGTM. Merged this.
* Build is passing - https://github.com/cnti-testcatalog/testsuite/actions/runs/8342423721

**Pull request merged**